static/js/components/email/EmailCompositionDialog.js

```diff
@@ -38,12 +37,18 @@ const EmailCompositionDialog = (props: EmailDialogProps) => {
     updateEmailEdit,
     open,
     email: { inputs, validationErrors, fetchStatus },
-    searchkit,
     sendEmail,
+    title,
+    children
   } = props;
 
+  let subHeading;
+  if (children) {
+    subHeading = <h5 className="sub-heading">{ children }</h5>;
+  }
```

Comment:
what do you think about pulling this out into a little helper method? so that in the `JSX` below we can just call

```js
{ this.subHeading() }
```